This hacks around a bug in reload I have not yet debugged up
with -m32 -mcpu=ultrasparc and in -m64 as well.

--- gcc/config/sparc/sparc.h.jj	Tue May 23 14:57:57 2000
+++ gcc/config/sparc/sparc.h	Thu May 25 16:54:36 2000
@@ -1029,13 +1029,18 @@ do								\
       fixed_regs[5] = 1;					\
     else if (TARGET_ARCH64 && fixed_regs[5] == 2)		\
       fixed_regs[5] = 0;					\
-    if (! TARGET_V9)						\
+    /* FIXME: if (! TARGET_V9) */				\
+    if (1)							\
       {								\
 	int regno;						\
 	for (regno = SPARC_FIRST_V9_FP_REG;			\
 	     regno <= SPARC_LAST_V9_FP_REG;			\
 	     regno++)						\
 	  fixed_regs[regno] = 1;				\
+      }								\
+    if (! TARGET_V9)						\
+      {								\
+	int regno;						\
 	/* %fcc0 is used by v8 and v9.  */			\
 	for (regno = SPARC_FIRST_V9_FCC_REG + 1;		\
 	     regno <= SPARC_LAST_V9_FCC_REG;			\
